x4.md

```diff
@@ -54,325 +49,93 @@ The protocol should be a reflection of our principles.
 
-The fact that anyone can participate makes the public chats voulnerable to spam attacks. Also, there are no moderators of these chats.
+The well-defined format also makes it possible to support multiple types of conversations as well as multiple types of content.
 
-## Sending
+Please refer to [Initial Message Payload Specification](x8.md) for more details.
 
-1. Calculate a symmetric key using [`shh_generateSymKeyFromPassword`](https://github.com/ethereum/go-ethereum/wiki/Whisper-v6-RPC-API#shh_generatesymkeyfrompassword) passing a public chat name as a string and save the result to `symKeyID`
-2. Call [`shh_post`](https://github.com/ethereum/go-ethereum/wiki/Whisper-v6-RPC-API#shh_post) with the following settings:
-   1. `symKeyID` MUST be set to `symKeyID`
-   2. `sig` MUST be set to `sigKeyID`
-   3. `ttl` MUST be at least `10` (it is in seconds)
-   4. `topic` MUST be set accordingly to [Topic](#topic) section and hex-encoded,
-   5. `payload` MUST be a hex-encoded string,
-   6. `powTime` MAY be arbitrary but should be enough to perform proof-of-work
-   7. `powTarget` MUST be equal or lower than `0.002`.
+# Security layer
 
-Learn more following [Whisper V6 RPC API](https://github.com/ethereum/go-ethereum/wiki/Whisper-v6-RPC-API).
+Security layer provides various cryptographical properties to the Status Protocol:
+* encryption
+* assurance of the origin
+* forward secrecy
 
-## Receiving
+Encryption makes sure that only author and the recipient can read the content of the message.
 
-Receiving public messages depends on Whisper filters idea. Upon receiving, messages are first matched by a topic and then trying to be decrypted using a symmetric key.
+Assurance of the origin using a digital signature ensures that the message was actually sent by the owner of a given public key and the message has not been changed induring the transport over the wire.
 
-1. Calculate a symmetric key using [`shh_generateSymKeyFromPassword`](https://github.com/ethereum/go-ethereum/wiki/Whisper-v6-RPC-API#shh_generatesymkeyfrompassword) passing public chat name as a string and save the result to `symKeyID`
-2. Call [`shh_subscribe`](https://github.com/ethereum/go-ethereum/wiki/Whisper-v6-RPC-API#shh_subscribe) with criteria:
-   1. `minPow` MUST be at least `0.002`
-   2. `topics` MUST be list of hex-encoded topics you expect messages to receive from (follow [Topic](#topic) section)
-   3. `allowP2P` MUST be set to `true` if offline messages are supported, otherwise can be `false`.
+Note: exchange and verification of the public keys is not a trivial problem. Read more about this problem in [Initial Trust Establishment Specification](x5.md).
```

Comment:
I'd phrase this more neutrally, something like

"Initial trust establishment is detailed in the following specification"

Alt

"This <key exchange etc> assumes trust has already been established